packages/react/slot/src/Slot.tsx

```diff
@@ -88,7 +88,10 @@ function mergeProps(slotProps: AnyProps, childProps: AnyProps) {
     const isHandler = /^on[A-Z]/.test(propName);
     // if it's a handler, modify the override by composing the base handler
     if (isHandler) {
-      overrideProps[propName] = composeHandlers(childPropValue, slotPropValue);
+      overrideProps[propName] = (...args: unknown[]) => {
```

Comment:
> That should be up to the components that are being composed together. I've added a story to show that events can still be prevented correctly.

Yeah I think you're actually right about that, and with that new signature, it does take care of the original issue someone had with these event compositions not receiving all params when it's not a native event handler.

:+1: